### PR TITLE
Sync e2e_node approvers with sig-node approvers

### PR DESCRIPTION
**Pull request by derekwaynecarr**
Synchronizes with approvers in main `kubernetes/kubernetes` repository for `/test/e2e_node`.

### CODE REVIEW

**Comment by derekwaynecarr** [comment]
Synchronizing the set of approvers based on experience from https://github.com/kubernetes/test-infra/pull/15609

File: jobs/e2e_node/OWNERS [hunk 1]
```diff
@@ -4,3 +4,8 @@ approvers:
 - Random-Liu
 - yguo0905
 - yujuhong
+- tallclair
+- derekwaynecarr
+- ConnorDoyle
+- klueska
+- dchen1107
```
Comment (yujuhong) [inline jobs/e2e_node/OWNERS]: Is this the list of OWNERS for the kubelet package or all sub-packages under it? It doesn't seem like either...

Comment (derekwaynecarr) [reply jobs/e2e_node/OWNERS]: @yujuhong I didnt remove any existing right, but just added the names from the /test/e2e_node/OWNERS listing here:
https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/OWNERS

**Pull request merged**